### PR TITLE
Installed and Configured DaisyUI

### PR DESCRIPTION
**Pull request by ValantineSuh**

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -1,9 +1,9 @@
 /** @type {import('tailwindcss').Config} */
-export default {
+module.exports = {
```
Comment (stephane-segning) [inline tailwind.config.js:2]: Why changing from `export default` to `module.exports` ?

File: src/app.scss [hunk 2]
```diff
@@ -1,3 +1,4 @@
 @import 'tailwindcss/base';
 @import 'tailwindcss/components';
-@import 'tailwindcss/utilities';
\ No newline at end of file
+@import 'tailwindcss/utilities';
+@import 'daisyui';
```
Comment (stephane-segning) [inline src/app.scss:4]: Why importing daisyUid here?

**Review by stephane-segning — CHANGES_REQUESTED** [review]

File: src/app.scss (hunk 2 above)
Comment (ValantineSuh) [reply src/app.scss:4]: Importing DaisyUI here ensures that its components and utilities are included in our project and properly integrated with Tailwind CSS.

Comment (stephane-segning) [reply src/app.scss:4]: This is not true. Check again how to install it

File: tailwind.config.js (hunk 1 above)
Comment (stephane-segning) [reply tailwind.config.js]: Cancel that change please. It'll lead to issues

Comment (ValantineSuh) [reply tailwind.config.js]: Sir, the change has already been made in the last commit

File: src/app.scss (hunk 2 above)
Comment (ValantineSuh) [reply src/app.scss:4]: On it

**Review by stephane-segning — CHANGES_REQUESTED** [review]
Already mentioned in comments

**Pull request closed without merging**